content/en/user-guide/ci/circle-ci/index.md

```diff
@@ -3,77 +3,444 @@ title: "CircleCI"
+      - store_artifact:
+          key: ls-state
+          paths: ls-state.zip
+...
+  localstack-load-state:
+    executor: localstack/default
+    steps:
+      ...
+      # LocalStack already running
+      - attach_workspace:
+          at: .
       - run:
-          name: Start LocalStack
+          name: Import state
           command: |
-            pip3 install localstack awscli-local[ver1]
-            docker pull localstack/localstack
-            localstack start -d                     
+            test -f ls-state.zip && localstack state import ls-state.zip
+...
+  workflows:
+  localstack-build:
+    jobs:
+      - localstack-save-state
+      - localstack-load-state
+```
+More information about state import and export [here](/user-guide/state-management/export-import-state).
+
+#### Cache
+To preserve state between workflow runs, you can take leverage of CircleCI's caching too.
+This strategy will persist LocalStack's state for every workflow re-runs, but not for different workflows.
+
+```yaml
+...
+jobs:
+  localstack-update-state:
+    executor: localstack/default
+    steps:
+      ...
+      # LocalStack already running
+      # Let's restore previous workflow run's LocalStack state
+      - restore_cache:
+          # Use latest "ls-state" prefixed cache
+          key: ls-state-
+      - run:
+          name: Import state
+          command: test -f ls-state.zip && localstack state import ls-state.zip
+      ...
+      # Infrastructure had been updated
+      # Let's update cached LocalStack state
+      - run:
+          name: Export state
+          command: localstack state export ls-state.zip
+      - save_cache:
+          key: ls-state-{{checksum ls-state.zip}}
+          paths: ls-state.zip
+  ...
+  localstack-do-work:
+    executor: localstack/default
+    steps:
+      # LocalStack already running
+      - restore_cache:
+          # Use latest "ls-state" prefixed cache
+          key: ls-state-
+      - run:
+          name: Import state
+          command: test -f ls-state.zip && localstack state import ls-state.zip
+      ...
+
+
+# Example workflows
+workflows:
+  localstack-build:
+    jobs:
+      - localstack-update-state
+      - localstack-do-work
+      ...
+```
+More information about state management [here](/user-guide/state-management/export-import-state).
 
-            echo "Waiting for LocalStack startup..."  
-            localstack wait -t 30                     
-            echo "Startup complete"
-            
+#### Cloud Pods
+Cloud pods providing an easy solution to persist LocalStack's state, even between workflows or projects.
+
+Find more information about cloud pods [here](/user-guide/state-management/cloud-pods/).
+
+##### Multiple projects
+Update or create the cloud pod in it's own project (ie in a separate Infrastructure as Code repo), this would create a base/"golden" cloud pod, which you can use in the future without any configuration or deployment.
```

Comment:
Golden Cloud Pod is still a loosely defined term used internally. Lets not use it on public-facing docs